configure

```diff
@@ -1428,6 +1458,10 @@ if options.prefix:
 
 config = '\n'.join(map('='.join, config.iteritems())) + '\n'
 
+setup_bin_overrides()
+if bin_override:
+  config = 'export PATH:=' + bin_override + ':${PATH}\n' + config
```

Comment:
Out of curiosity, why `${PATH}` and not `$(PATH)`?

Reply:
It's what was on the stack overflow answer. 😝 Will change if I can